clarity-ext-scripts/clarity_ext_scripts/covid/validate_sample_creation_list.py

```diff
@@ -87,9 +87,13 @@ def execute(self):
                     logger.warn("Using testing org. Service request ID faked: {}".format(
                         service_request_id))
                 else:
-                    response = client.search_for_service_request(
-                        org_uri, barcode)
-                    service_request_id = response["resource"]["id"]
+                    try:
+                        response = client.search_for_service_request(
+                            org_uri, barcode)
+                        service_request_id = response["resource"]["id"]
+                    except OrganizationReferralCodeNotFound as e:
+                        self.usage_error_defer(("Cannot find service_request_id "
```

Comment:
I think those parentheses are not required, i.e. `usage_error_defer((` => `usage_error_defer`. But it doesn't matter either.

But `usage_error_defer`s message should not contain variable data. It should be the "category" of the error. So in this case I would do:

```self.usage_error_defer("Cannot find service_request_id for (org, barcode)", "{}/{}".format(org_uri, barcode))
```

When the error is thrown in the end of the handler, the message to the user will read something like e.g:

```
Cannot find service_request_id for (org, barcode): [org1/barcode1, org2/barcode2]
```